src/test/PDFTest/CrudPDFServiceTest.java

```diff
@@ -86,26 +86,7 @@ public void uploadFormTest() {
   //    uploadTestAnnotatedFormPDF();
   //  }
 
-  @Test
```

Comment:
why are we just removing this test? Don't we need the /upload function for profile pictures?

Reply:
Bump on this